src/test/resources/alma-fix/990001412590206441.json

```diff
@@ -68,6 +68,10 @@
     "id" : "http://digitale-objekte.hbz-nrw.de/storage2/2019/04/06/file_4/8295425.pdf"
   } ],
   "inCollection" : [ {
+    "id" : "http://lobid.org/organisations/DE-655",
```

Comment:
The trailing `#!` is missing, right?

Reply:
Forgot to run the tests again. Fixed now.